src/Debug/PathFinding.cpp

```diff
@@ -37,6 +37,8 @@ PathFinding::PathFinding()
 
 void PathFinding::Draw(Game& game)
 {
+	// TODO(falseilyu): #626 Get a better way to deal with unused parameter warning.
+	std::ignore = game;
```

Comment:
[[maybe_unused]] in front of the parameter.

Reply:
Thanks